### DISCUSSION TITLE
Parameter set for LMO cathode

### DISCUSSION THREAD

**Discussion by Thiyaga2025** [pybamm-team/PyBaMM#4497] [Q&A] [answered]
team Pybamm,

I am trying to model NMC/LMO(LiMnO2) blend cathode. Looking for properties of LMO. which parameter set is to be used for LMO.

thanks.

**Comment by Thiyaga2025** [comment]
any update on my query?

**Reply by ALMA7867** [reply]
You can check on liiondb database website.

**Reply by Thiyaga2025** [reply]
@ALMA7867 ,

thanks for the response. unable to open the portal. can you or anyone please send me the correct URL.

regards...

**Reply by agriyakhetarpal** [reply, marked as answer]
The LiionDB database can be accessed via a Jupyter notebook interface: https://github.com/ndrewwang/liiondb#-using-liiondb-with-python

**Reply by agriyakhetarpal** [reply]
There are a bunch of open issues there, though, so I'm unsure if it works or can be made to work without much hassle.

**Reply by Thiyaga2025** [reply]
@agriyakhetarpal , thanks, will check.
